src/utils/env/index.mjs

```diff
@@ -145,6 +146,18 @@ export const getEnvelopeEnv = async ({ api, context = 'dev', env, key = '', scop
 
   const accountEnv = formatEnvelopeData({ context, envelopeItems: accountEnvelopeItems, scope, source: 'account' })
   const siteEnv = formatEnvelopeData({ context, envelopeItems: siteEnvelopeItems, scope, source: 'ui' })
+
+  if (raw) {
+    const entries = Object.entries({ ...accountEnv, ...siteEnv })
```

Comment:
Is it correct that only account and site env vars, and not general / addons / configFile env vars, need to be included as siteEnv for the functionsConfig? That's how it seems to be before -- only `get(siteData, 'build_settings.env')` was getting passed to functionsConfig.

Reply:
Yes, I think that's right.

Reply:
OK cool! then I think this would fix a bug where account-level env vars weren't getting applied to functions if a user deployed with this method.